### PR TITLE
feat: [sc-11192] ✨ [Frontend] Spark product hub items

### PR DESCRIPTION
**Pull request by marcinciarka**
Added product hub integration for:
- Multiply & Borrow:
  - wstETH/DAI
  - rETH/DAI
  - ETH/DAI
  - sDAI/ETH
- Earn
  - wstETH/ETH
  - rETH/ETH
  
Additionally all of the pages are viewable and the proper icons are present.

How to test:
1. You need to `yarn link` the addresses package from the `feature/spark-protocol` on earn-sc for this to work (or probably install the alpha version specified)

### CODE REVIEW

File: blockchain/contracts/arbitrum.ts [hunk 1]
```diff
@@ -147,13 +147,9 @@ export const arbitrumContracts: MainnetContracts = {
...
   aaveV3Oracle: contractDesc(aaveV3Oracle, arbitrum.aave.v3.Oracle),
   aaveV3PoolDataProvider: contractDesc(aaveV3PoolDataProvider, arbitrum.aave.v3.PoolDataProvider),
-  sparkV3Pool: contractDesc(
-    sparkV3Pool,
-    arbitrum.aave.v3.LendingPool,
-    AAVE_V3_POOL_GENESIS_MAINNET,
-  ),
-  sparkV3Oracle: contractDesc(sparkV3Oracle, arbitrum.aave.v3.Oracle),
-  sparkV3PoolDataProvider: contractDesc(sparkV3PoolDataProvider, arbitrum.aave.v3.PoolDataProvider),
+  sparkV3Pool: contractDesc(sparkV3Pool, arbitrum.spark.LendingPool!),
```
Comment (jakubswierczek) [inline blockchain/contracts/arbitrum.ts:150]: do we need the exclamation mark here?

File: handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    memoizedTokensData(networkName as SparkV3Networks, tickers),
+  )
+
+  const yieldsPromisesMap: Record<
+    string,
+    (risk: RiskRatio, fields: FilterYieldFieldsType[]) => Promise<AaveLikeYieldsResponse>
+  > = {
+    'WSTETH/ETH': emptyYields,
+    'CBETH/ETH': emptyYields,
+    'RETH/ETH': emptyYields,
```
Comment (jakubswierczek) [inline handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts:98]: are all the pair available on Spark?

File: handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        )![primaryToken]
+        const tokensAddresses = getNetworkContracts(NetworkIds.MAINNET).tokens
+
+        return {
+          ...product,
+          primaryTokenAddress: tokensAddresses[primaryToken].address,
+          secondaryTokenAddress: tokensAddresses[secondaryToken].address,
+          maxMultiply:
+            product.label === 'WSTETH/ETH'
+              ? wstethRiskRatio.multiple.toString()
```
Comment (jakubswierczek) [inline handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts:147]: it's a hack. Do we have emode on the Spark?

File: handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts (hunk 2 above)
Comment (marcinciarka) [reply handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts:97]: Fixed.

File: handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts (hunk 3 above)
Comment (marcinciarka) [reply handlers/product-hub/update-handlers/sparkV3/sparkV3Handler.ts:160]: Fixed.

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**